### PR TITLE
Remove delta-resistance

### PR DESCRIPTION
**Pull request by tjknoth**
Updates due to jitx-client PR 4856

### CODE REVIEW

File: slm.toml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 name = "ocdb"
-version = "3.12.0"
+version = "3.14.0"
```
Comment (bhusang) [inline slm.toml:2]: This should probably some later version? 3.15?

**Review by bhusang — APPROVED** [review]

**Comment by bhusang** [comment]
This needs the docker compose command to be updated. You might need to merge main or dev back into this branch? @jwatson0 might have a better idea.

**Comment by plasmajitx** [comment]
btw, PR should be to `dev` I think.  `dev` corresponds to `develop` on jitx-client, `main` is latest public release.

**Comment by tjknoth** [comment]
> btw, PR should be to `dev` I think. `dev` corresponds to `develop` on jitx-client, `main` is latest public release.

oops -- will close and reopen a new one

**Pull request closed without merging**